DiffusionPreprocessing/scripts/eddy_postproc.sh

```diff
@@ -102,10 +120,12 @@ if [ ! $GdCoeffs = "NONE" ] ; then
     ${FSLDIR}/bin/imrm ${datadir}/grad_dev_?
     ${FSLDIR}/bin/imrm ${datadir}/trilinear
     ${FSLDIR}/bin/imrm ${datadir}/data_warped_vol1
-    
+    ${FSLDIR}/bin/imrm ${datadir}/data_warped_dilated
+
     #Keep the original warped data and warp fields
     mkdir -p ${datadir}/warped
     ${FSLDIR}/bin/immv ${datadir}/data_warped ${datadir}/warped
+    ${FSLDIR}/bin/immv ${datadir}/fov_mask_warped ${datadir}/warped
     ${FSLDIR}/bin/immv ${datadir}/fullWarp ${datadir}/warped
     ${FSLDIR}/bin/immv ${datadir}/fullWarp_abs ${datadir}/warped
 fi
```

Comment:
With the application of the fov_mask, we shouldn't have any negative values due to the (GDC) spline interpolation, so can we update the following comment to the following:
`#Remove negative intensity values (from eddy) from final data`

Reply:
Done

Reply:
Done